mne/source_space.py

```diff
@@ -1348,10 +1353,10 @@ def setup_source_space(subject, fname=True, spacing='oct6', surface='white',
 
 
 @verbose
-def setup_volume_source_space(subject, fname=None, pos=5.0, mri=None,
+def setup_volume_source_space(subject=None, fname=None, pos=5.0, mri=None,
```

Comment:
sorry to question this but why not having just subject and deprecate the mri parameter?

if mri is always T1.mgz maybe we don't need the mri param?

@Eric89GXL wdyt?